.eslintrc.js

```diff
@@ -11,5 +14,6 @@ module.exports = {
         "**/*.config.js",
       ],
     }],
+    "react/jsx-one-expression-per-line": ["warn"],
```

Comment:
What is the reason of adding this line?

Reply:
```
<p>
                  The verification email has been sent to
                  {email}.
                  <br />
                  Please follow the instructions from the email to complete a signup process.
                </p>

Reply:
`{email}.`
To be able to put a dot here, and not to transfer it to a new line (in this case, a space will be added before the dot, this is undesirable)

Reply:
@SobakaSlava that's correct. We decided to use warn for now unless you see a better solution. By default as @Mar1nka mentioned it forces to put {email} on the separate line which adds a space before the dot